bitnami/nginx/README.md

```diff
@@ -77,6 +77,7 @@ The following tables lists the configurable parameters of the NGINX Open Source
 | `extraVolumeMounts`                        | Array of extra volume mounts to be added to the deployment. Normally used with `extraVolumes`| `nil`                                                        |
 | `replicaCount`                             | Number of replicas to deploy                                                                 | `1`                                                          |
 | `containerPort`                            | Deployment Container Port                                                                    | `8080`                                                       |
+| `containerTlsPort`                         | Optional Deployment Container Tls Port                                                       | `nil`                                                        |
```

Comment:
I would remove the "Optional" here as we are describing the purpose of the parameter, if it's optional or not it depends on the configuration, by default is not enabled.